src/databricks/labs/remorph/reconcile/recon_config.py

```diff
@@ -11,6 +12,10 @@
 logger = logging.getLogger(__name__)
 
 
+class TableThresholdBoundsException(ValueError):
+    """Raise the error when the bounds for table threshold are invalid"""
```

Comment:
Is this custom exception being checked specifically somewhere in the code? If not, just using ValueError would be sufficient.

Reply:
This would actually giver better understanding of the error